docs/troubleshooting-guide/models.md

```diff
@@ -24,20 +24,24 @@ If your changes to a model's metadata or underlying question aren't showing up:
 
 ## Model performance is poor
 
-1. Optimize the underlying saved question or SQL query.
+1. Use model persistence.
+    
+    - Metabase can [write back to your data warehouse](data-modeling/models#model-caching), so if you enable model caching, Metabase will read a pre-computed and materialized version of the model directly from the database instead of running the query from scratch
```

Comment:
```suggestion
    - Metabase can [write back to your data warehouse](data-modeling/models#model-caching). If you enable model caching, Metabase will read a pre-computed and materialized version of the model directly from the database instead of running the query from scratch.
```
Should we mention that model caching is only available for some data sources?

Reply:
agree!